packages/logger/logger.0.4.2/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+
+name: "logger-p5"
```

Comment:
The `name` and `version` fields are redundant, `opam` will replace them with what infers from the path.
Moreover, is the name `logger` (as in the path) or `logger-p5` (as here)?

Reply:
I made a transition to `logger-p5`. Thanks